.github/workflows/test_docker.yml

```diff
@@ -19,16 +21,19 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 5.0.x
+        dotnet-version: |
```

Comment:
Does it mean we'll run with each of them ?

Reply:
no. its just what's required to run them. 3.1 is required because our test-server is ASP.Net based as of today and it requires 3.1.